src/Sounder.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2023 Matthew Taylor
+*/
+//
+
+#include "Sounder.h"
+#include "picoPlatform.h"
+#include "KiwiBoardFirmware_menu.h"
+
+PicoPlatform *hardware;
+Beep beepobj[2];
+bool BeepHandler::soundset = 0;
+
+
+void BeepHandler::beep_activate(int tone) {
+
+    int beep = tone;
+    
+    if(soundset) {
+
+        beepobj[beep].beep_activate = 1;
+        Serial.println("Beep activated");
+
+    }
+
+}
+
+// Initialize the end of cycle tone
+void BeepHandler::initTone1() {
```

Comment:
Both of these init methods can be moved into the constructor (That is what the constructor is meant to do, setup any instance data required for the object, which is what these are doing).